feature-group-definitions/gradients.yml

```
@@ -0,0 +1,50 @@
+spec: https://drafts.csswg.org/css-images-4/#gradients
+caniuse:
+  - css-gradients
+  - css-repeating-gradients
+  - css-conic-gradients
+compat_features:
+  # Background-image prop
+  - css.properties.background-image.gradients
+  - css.properties.background-image.multiple_backgrounds
+  # CSS gradient image type
+  - css.types.image.gradient
+  # Linear gradients
+  - css.types.image.gradient.linear-gradient
+  - css.types.image.gradient.linear-gradient.doubleposition
+  - css.types.image.gradient.linear-gradient.hue_interpolation_method
+  - css.types.image.gradient.linear-gradient.interpolation_color_space
```

Firefox hasn't implemented these yet. Since they seem to be unique in the bunch for being unsupported, I think we ought to omit them as well. (Commenting them out to show that this is an intentional choice, though I suppose we could also just delete those lines.)

```suggestion
  # - css.types.image.gradient.linear-gradient.hue_interpolation_method
  # - css.types.image.gradient.linear-gradient.interpolation_color_space
```